Tribler/Core/Modules/restapi/rest_manager.py

```diff
@@ -30,7 +30,7 @@ def start(self):
         self.root_endpoint = RootEndpoint(self.session)
         site = server.Site(resource=self.root_endpoint)
         site.requestFactory = RESTRequest
-        self.site = reactor.listenTCP(self.session.get_http_api_port(), site)
+        self.site = reactor.listenTCP(self.session.get_http_api_port(), site, interface="127.0.0.1")
```

Comment:
I think right now we should hold off on that, at least until the web api has some sort of security mechanism. Users may not understand that the web api is unprotected..